pvlib/test/test_bifacial.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import pandas as pd
+from datetime import datetime
+from pvlib.bifacial import pvfactors_timeseries
+from conftest import requires_pvfactors
+
+
+@requires_pvfactors
+def test_pvfactors_timeseries():
+    """ Test that pvfactors is functional, using the TLDR section inputs of the
+    package github repo README.md file"""
+
+    # Create some inputs
+    timestamps = pd.DatetimeIndex([datetime(2017, 8, 31, 11),
+                                   datetime(2017, 8, 31, 12)])
+    solar_zenith = [20., 10.]
+    solar_azimuth = [110., 140.]
+    surface_tilt = [10., 0.]
+    surface_azimuth = [90., 90.]
+    dni = [1000., 300.]
+    dhi = [50., 500.]
+    gcr = 0.4
+    pvrow_height = 1.75
+    pvrow_width = 2.44
+    albedo = 0.2
+    n_pvrows = 3
+    index_observed_pvrow = 1
+    rho_front_pvrow = 0.03
+    rho_back_pvrow = 0.05
+    horizon_band_angle = 15.
+
+    # Expected values
+    expected_ipoa_front = [1034.96216923, 795.4423259]
+    expected_ipoa_back = [92.11871485, 70.39404124]
+    tolerance = 1e-6
+
+    # Test serial calculations
+    ipoa_front, ipoa_back, _ = pvfactors_timeseries(
```

Comment:
Why not test the `df_registries` value?

Reply:
Very good point: 
- it is partly and indirectly tested as ``ipoa_front`` and ``ipoa_back`` do come from ``df_registries`` (see [here](https://github.com/pvlib/pvlib-python/blob/9829b810ebedfc22172e51d499d1b271f766a537/pvlib/bifacial.py#L134-L136))
- but on a more general note, I guess my intent was to keep things simple. We could see ``df_registries`` as a "master document" containing very detailed results (maybe mostly unused too: see example [here](https://sunpower.github.io/pvfactors/developer/pvfactors_demo.html#Surface-Registry) or also [here](https://sunpower.github.io/pvfactors/developer/pvfactors_demo.html#Calculate-radiosities-for-single-time-point-and-using-isotropic-diffuse-sky-dome-approach)). And for simplicity, I wanted to extract and test what I thought most users would be interested in, which are the aggregated front and back surface POA irradiance values, while still giving users the option of looking at the details.

I can add some tests for other outputs in ``df_registries``, let me know! (I might grab some tests from the pvfactors repo)
Here is what the ``df_registries`` expected values would look like for the given test inputs we have so far:
![image](https://user-images.githubusercontent.com/12040910/50203061-9ff99f80-03a3-11e9-9730-92afe74eae1c.png)
Is it okay to use csv files to store test data in the ``pvlib`` repo?

Reply:
That makes sense. I don't have a strong opinion on whether or not this should be tested in pvlib.

>  Is it okay to use csv files to store test data in the pvlib repo?

Yes, see [pvlib/data](https://github.com/pvlib/pvlib-python/tree/master/pvlib/data). Your example is a good size for a test file.

Reply:
Thanks @wholmgren , I don't think that there is a strong need to implement this here. I tested a lot of different cases in the pvfactors repo, so the probability of getting an issue here should be low. But don't hesitate to let me know if you want it at any point

Reply:
I think we're ok without testing the `df_registries` output.  In pvlib we're confirming that `pvfactors_timeseries` is returning values consistent with separate `pvfactors` calculations and its enough to test `ipoa_front` and `ipoa_back`.